aws-predictions/src/androidTest/java/com/amplifyframework/predictions/aws/AWSPredictionsTranslateTest.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.aws;
+
+import android.content.Context;
+
+import com.amplifyframework.predictions.PredictionsCategory;
+import com.amplifyframework.predictions.PredictionsException;
+import com.amplifyframework.predictions.aws.test.R;
+import com.amplifyframework.predictions.models.LanguageType;
+import com.amplifyframework.predictions.options.TranslateTextOptions;
+import com.amplifyframework.predictions.result.TranslateTextResult;
+import com.amplifyframework.testutils.sync.SynchronousMobileClient;
+import com.amplifyframework.testutils.sync.SynchronousPredictions;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static androidx.test.core.app.ApplicationProvider.getApplicationContext;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * Tests that Predictions translate delivers a non-null result
+ * for valid input.
+ */
+public final class AWSPredictionsTranslateTest {
+
+    private SynchronousPredictions predictions;
+
+    /**
+     * Initialize mobile client singleton.
+     * @throws Exception if mobile client initialization fails
+     */
+    @BeforeClass
+    public static void setUpOnce() throws Exception {
+        SynchronousMobileClient.instance().initialize();
```

Comment:
Honestly my main thing is that this second setup method takes ~10 lines and the reader has to find both of them, instead of just 1 line in the setup. But, I hear what you're saying.

Reply:
I will actually do it the way you suggested, because this reminder isn't really helpful considering that it is not immediately fixable. Even with Amplify Auth category implementation, we will still be relying on AWS Mobile Client for some time.